backend/spring-boot/src/main/java/org/bootstrapbugz/backend/user/service/impl/UserServiceImpl.java

```diff
@@ -24,10 +24,13 @@ public UserServiceImpl(UserRepository userRepository, MessageService messageServ
   }
 
   @Override
-  public List<UserDTO> findAll(Pageable pageable) {
-    return userRepository.findAll(pageable).stream()
-        .map(UserMapper.INSTANCE::userToSimpleUserDTO)
-        .toList();
+  public PageableDTO<UserDTO> findAll(Pageable pageable) {
+    final var users =
+        userRepository.findAll(pageable).stream()
+            .map(UserMapper.INSTANCE::userToSimpleUserDTO)
+            .toList();
+    final var total = userRepository.count();
+    return new PageableDTO<>(users, total, pageable.getPageNumber(), pageable.getPageSize());
```

Comment:
Looks good overall, but maybe reconsider if there's a need for _pageNumber_ and _pageSize_ fields in PageableDTO, since they are always equal to the RequestParams _page_ and _limit_ that are being sent from the frontend